MNE/disp3D/3DObjects/bem/bemsurfacetreeitem.cpp

```diff
@@ -97,6 +97,8 @@ BemSurfaceTreeItem::BemSurfaceTreeItem(int iType, const QString& text)
 
 BemSurfaceTreeItem::~BemSurfaceTreeItem()
 {
+    setVisible(false);
```

Comment:
is this call necessary? I mean the Object is destroyed anyway, why calling this member method?